oval-schemas/windows-definitions-schema.xsd

```diff
@@ -1485,7 +1485,17 @@
       <xsd:element name="ntuser_object" substitutionGroup="oval-def:object">
             <xsd:annotation>
                   <xsd:documentation>The ntuser_object element is used to specify which metadata should be collected from a Windows ntuser.dat file.  Each object extends the standard ObjectType as defined in the oval-definitions-schema and one should refer to the ObjectType description for more information. The common set element allows complex objects to be created using filters and set logic. Again, please refer to the description of the set element in the oval-definitions-schema.</xsd:documentation>
-                  <xsd:appinfo>
+                  <xsd:documentation>To ensure consistent results across OVAL interpreters should comply with the following:</xsd:documentation>
+				  <xsd:documentation>1.  Only include ntuser.dat files from 'human' accounts that have interactively logged into the system, with the exception of the 'Default' user if the 'include_default' behavior is true.</xsd:documentation>
+				  <xsd:documentation>2.  Create a 'does not exist' item for each ntuser.dat file that does contain the required registry key.</xsd:documentation>
+				  <xsd:documentation></xsd:documentation>
+				  <xsd:documentation>For requirement #1, OVAL interpreters are at their discretion to determine which ntuser.dat files have been logged into interactively, but the following is one method to determine this information.</xsd:documentation>
+				  <xsd:documentation>a.  Obtain a list of user profiles from the registry key:  'HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion\ProfileList'</xsd:documentation>
```

Comment:
Couldn't you just check that the user has logged on at some point?  Using whatever method is used to determine last_logon for the various user tests?  Might be simpler, since every interpreter already has to be able to collect that.

Reply:
Yeah, I guess I do remember that you had to query the domain controller for domain accounts.  But, doesn't that give you the last time the user logged onto the domain?  We want the last time the user logged into the machine -- which is what you're capturing using a trick.  I wonder if that would capture logins using WS-Man/WinRM.  I don't have access to the method we used to capture the last_logon data, I'd need @maxullman or @A-Biggs to chime in.

Reply:
If we could accurately determine the last_login relative to the machine being logged in, that would be a big win, and IMHO would warrant adding a 'last_login' attribute to the ntuser_item, as currently we just have last_modified, which can be inaccurate, as Windows updates sometimes arbitrarily update all ntuser.dat files on the system, making the file modified date worthless.